sources/_sector/dex/trades/blast/_sources.yml

```diff
@@ -17,6 +17,12 @@ sources:
       - name: ThrusterFactory_Point_3_Fee_evt_PairCreated
       - name: ThrusterPair_V2_1_Fee_evt_Swap
       - name: ThrusterFactory_1_Fee_evt_PairCreated
+  - name: bladeswap_blast
```

Comment:
With extra sources introduced (which I may miss), we may implement the `bladeswap.trades` in a more proper way.